internal/integration/actions.go

```diff
@@ -170,7 +172,7 @@ func Exec(cmd string, args ...string) Action {
 func ExecWithExpectedOutput(want string, cmd string, args ...string) Action {
 	return func(t testing.TB, ic TestContext) {
 		Infof("Executing: %s %s", cmd, shellquote.Join(args...))
-		output, err := ftlexec.Capture(ic, ic.workDir, cmd, args...)
+		output, err := ftlexec.CaptureWithEnv(ic, ic.workDir, cmd, []string{"FTL_ENDPOINT=http://127.0.0.1:" + TestPort}, args...)
```

Comment:
Rather than having to do this, let's just set FTL_ENDPOINT in Run() with `t.Setenv()` then it's just set everywhere by default.